### PR TITLE
Alpha version - Message standardization (edit feature)

### LINKED ISSUE #99: Alpha version - Message standardization (edit feature)
Tweak formats to ensure consistency with previously standardized messages.

### PR DESCRIPTION
**Pull request by wkxcass**
Fixes #99 

Tweaked formats to ensure consistency with previously standardized messages.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -51,9 +51,9 @@ public class EditCommand extends Command {
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited successfully! Edited participant: %1$s";
+    public static final String MESSAGE_NOT_EDITED = "Provide at least one field to edit!";
```
Comment (wkxcass) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:55]: Made use of imperative mood here #CS2101 🔥

Comment (k-hian) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:55]: Could you standardise this language FindCommand (MESSAGE_NO_CRITERIA) as well?

**Review by k-hian — COMMENTED** [review]
LGTM, just the one extra message to standardise.

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (k-hian) [reply src/main/java/seedu/address/logic/commands/EditCommand.java:55]: Sorry, it seems like the change is made in the other PR. You can ignore this suggestion.

**Pull request merged**